source/components/ArchivesList.js

```diff
@@ -167,7 +159,7 @@ class ArchivesList extends Component {
         this.rehydrationComplete = false;
     }
 
-    componentWillReceiveProps(nextProps) {
+    UNSAFE_componentWillReceiveProps(nextProps) {
```

Comment:
Might be time for a refactor soon huh? 😅

Reply:
haha yeah :D